### PR TITLE
OpenAPI initial version

### PR DESCRIPTION
**Pull request by fcarelse**

### CODE REVIEW

**Comment by fcarelse** [comment]
Seems ok for a initial version of the OpenAPI document.

**Review by fxcie — APPROVED** [review]
Looks good to me

**Review by fxcie — APPROVED** [review]
Approved

File: openapi.yml [hunk 1]
```diff
@@ -0,0 +1,150 @@
+openapi: 3.0.3
+info:
+  title: Todolist Ireland
+  description: Todolist service
+  version: 0.1.1
+servers:
+  - url: https://todolist.ie/
+    description: Web server
+  - url: https://dev.todolist.ie/
+    description: Staging Web server
```
Comment (Kivylius) [inline openapi.yml]: this should say: "Dev Web Server" else it should be stage.todolist.ie

File: openapi.yml [hunk 2]
```diff
@@ -0,0 +1,150 @@
+openapi: 3.0.3
+info:
+  title: Todolist Ireland
+  description: Todolist service
+  version: 0.1.1
+servers:
+  - url: https://todolist.ie/
+    description: Web server
```
Comment (Kivylius) [inline openapi.yml]: this should say probably say:  "Prod Web Server"

**Review by Kivylius — APPROVED** [review]
2 small NITs, but other then that LGTM.

**Pull request merged**